src/firefly/java/edu/caltech/ipac/util/expr/expressions.html

```diff
@@ -0,0 +1,228 @@
+<!--
+Copyright 2002 by Darius Bacon <darius@wry.me>
+Altered
+-->
+<!DOCTYPE html>
+
+<html>
```

Comment:
I am not sure this is the right place to put this file.  But, I can't think of a better place for it.